### PR TITLE
Test with Rails 4.2.8

### PR DESCRIPTION
**Pull request by n-rodriguez**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -53,11 +53,24 @@ gemfile:
...
   - gemfiles/rails_4.2.7.1.gemfile
+  - gemfiles/rails_4.2.8.gemfile
   - gemfiles/rails_5.0.0.gemfile
   - gemfiles/rails_5.0.0.1.gemfile
   - gemfiles/rails_5.0.1.gemfile
+  - gemfiles/rails_5.0.2.gemfile
 
 matrix:
+  include:
+    - rvm: 2.4.0
```
Comment (jonkessler) [inline .travis.yml]: How come we're only testing 4.2.8 and 5.0.x against 2.4.0?

Comment (n-rodriguez) [inline .travis.yml]: http://weblog.rubyonrails.org/2017/2/21/Rails-4-2-8-has-been-released/

**Comment by n-rodriguez** [comment]
Thank you! See you on Rails 5.1!

**Pull request merged**